### PR TITLE
added project toml

### PR DESCRIPTION
**Pull request by matbesancon**
Long overdue Project.toml on Clp, to better track dependencies, etc

### CODE REVIEW

**Comment by matbesancon** [comment]
@mtanneau maybe this before #63 ?

**Comment by mtanneau** [comment]
Closes #62.

I had added a `Project.toml`, in my MOI branch ; I'll just rebase once this is merged.

**Comment by matbesancon** [comment]
cc @blegat

**Comment by mlubin** [comment]
Also delete `REQUIRE`?

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name = "Clp"
+uuid = "5e71592d-7078-450d-8580-c595f51aa13b"
+authors = ["Mathieu Besançon <mathieu.besancon@gmail.com>"]
```
Comment (blegat) [inline Project.toml]: I would not include this field for consistency with other solver repo in JuliaOpt.

Comment (matbesancon) [reply Project.toml]: sorry hadn't seen it was still there, it got generated

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+repo = "https://github.com/JuliaOpt/Clp.jl.git"
+version = "0.6.3"
+
+[deps]
+BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```
Comment (blegat) [inline Project.toml]: We can remove it as we require Julia v1.0

**Comment by mlubin** [comment]
We should also drop 0.7 from travis and appveyor.

File: Project.toml (hunk 2 above)
Comment (matbesancon) [reply Project.toml]: it seems MPB requires Compat for its tests

Comment (matbesancon) [reply Project.toml]: Compat can be dropped when we don't support MPB

**Comment by matbesancon** [comment]
Removing Compat all together is for another time, since it requires removing MPB

**Comment by matbesancon** [comment]
all tests passing here

**Comment by blegat** [comment]
> Removing Compat all together is for another time, since it requires removing MPB

You can keep `Compat` only for the `test` target

**Comment by matbesancon** [comment]
Just did

**Pull request merged**